modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -618,6 +618,11 @@ export class Wallet implements IWallet {
       .send(filteredParams)
       .result();
 
+    response.txInfo?.unspents?.map((u) => {
+      u.value = BigInt(u.valueString);
+      return u;
+    });
```

Comment:
We have to be very careful here. We don't know where `response.txInfo` ends up precisely (mind you this is the generic code path for all utxo coins). If this value gets mixed up with regular `number` values or with a HTTP POST call, there will be problems.

Better do a conversion where it is needed.

Reply:
Moved conversion to signTransaction of abstractUtxoCoin and behaviour is coin specific now.